src/js/components/Form/ProjectPicker.jsx

```diff
@@ -0,0 +1,167 @@
+import PropTypes from 'prop-types'
+import React, { useContext, useEffect, useState } from 'react'
+
+import { Context } from '../../state'
+import { httpGet, httpRequest, requestOptions } from '../../utils'
+import { useTranslation } from 'react-i18next'
+
+function ProjectPicker({
+  disabled,
+  name,
+  onChange,
+  onError,
+  readOnly,
+  required,
+  value
+}) {
+  const [globalState, dispatch] = useContext(Context)
```

Comment:
Are we storing the `dispatch` function in our context?

Reply:
[Yes](https://github.com/AWeber-Imbi/imbi-ui/blob/82edd633647b86b4b0d5a7af398bc51f2a8ffef1/src/js/state.jsx#L108-L110). However, we often don't use it, so I think I can remove the variable